lib/vehicle_factory.rb

```diff
@@ -0,0 +1,18 @@
+class VehicleFactory
+    def create_vehicles(registrations = [])
+      vehicles = []
+      registrations.each do |registration|
```

Comment:
Nice work! You can also swap out `.each` for `.map` here and avoid the need for your `vehicles` array as an accumulator. But both solutions are good!